docs/dxp/latest/en/users-and-permissions/developer-guide/api/document-api-basics/resources/liferay-y6q4.zip/curl/User_GET_ById.sh

```diff
@@ -0,0 +1,3 @@
+curl \
+GET "http://localhost:8080/o/headless-admin-user/v1.0/user-accounts/${1}" \
```

Comment:
Since curl's default HTTP method is `GET`, we don't specify it in the command. So the second line should only have the endpoint URL.